docs/src/pages/components/data-grid/editing/editing.md

```diff
@@ -5,7 +5,123 @@ components: DataGrid, XGrid
 
 # Data Grid - Editing
 
-<p class="description">Data Grid will provide full support to create, read, update, and delete operations (CRUD).</p>
+<p class="description">The data grid has built-in edit capabilities that you can customize to your needs.</p>
+
+## Cell editing
+
+Cell editing allows to edit the value of one cell at a time.
+
+- set `editable` attribute of the column definition `GridColDef` object to `true` to allow editing cells of this column.
+
+```tsx
+<DataGrid columns={[{ field: 'name', editable: true }]} />
+```
+
+### Start Editing
+
+If a column is editable and if a cell has focus.
+
+- Press <kbd class="key">Enter</kbd> to switch to edit mode
+- Press <kbd class="key">Delete</kbd> to delete the value
+- Type any alpha key to start editing
+- Double click to switch to edit mode
+
+{{"demo": "pages/components/data-grid/editing/BasicEditingGrid.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Stop Editing
+
+- <kbd class="key">Escape</kbd> to rollback changes.
+- <kbd class="key">Tab</kbd> to save and goes to next cell.
+- <kbd class="key">Enter</kbd> to save and goes to next cell below.
+- Click outside the cell
+
+### Control cell editability
+
+In addition to the `editable` flag on columns, control which cell is editable using `isCellEditable` prop.
+
+_In this demo, `Name` is not editable, and only the rows with an even `Age` value are editable.
+We applied a green background for a better visibility._
```

Comment:
No we, [point 1.](https://www.notion.so/Technical-documentation-7e55b517ac2e489a9ddb6d0f6dd765de)

```suggestion
The editable cells have a green background for better visibility.
```